Rn okta364354 remove dev console (#1905)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/quickstart/add-user/index.md

Any authentication system needs to have a way of keeping track of the end users who are allowed to authenticate.

> When you use Okta for authentication, you don't need to implement your own user management solution. Creating and deleting users, configuring their authentication flows, and resetting their passwords, can all be done from the Admin Console UI. User profiles are stored securely for you in the Okta cloud.

Reviewer: ```suggestion
When you use Okta for authentication, you don't need to implement your own user management solution. Creating and deleting users, configuring their authentication flows, and resetting their passwords, can all be done from the Admin Console. User profiles are stored securely for you in the Okta cloud.
```

To create a new user:

> 1. From the Dashboard's side navigation, select **Directory** and then **People**

Reviewer: ```suggestion
1. From the Admin Console side navigation, select **Directory** and then **People**
```

4. For password, select "Set by user".
5. Click **Save**.

> As you’ll see later, in addition to manually creating a user in the Admin Console, you can also create users via API, and there are multiple ways of doing batch imports of users from existing user stores.

Reviewer: ```suggestion
In addition to manually creating a user in the Admin Console, you can also create users through the API, and there are multiple ways of doing batch imports of users from existing user stores.
```
